RMS-53640, RMS-53642

dterpal opened the pull request:
Added posts about adding tax rates and waiter teams to Api

dterpal committed:
Front.Api

Tax rates and waiter teams

zed220 commented on _posts/2022-07-25-waiter-teams.md:
```diff
@@ -0,0 +1,8 @@
+
+В API V8Preview2 был добавлен интерфейс бригады официантов [`IWaiterTeam`](https://iiko.github.io/front.api.sdk/v8/html/T_Resto_Front_Api_Data_Device_IWaiterTeam.htm).
+
+Также в API стали доступны методы [`GetWaiterTeams()`](https://iiko.github.io/front.api.sdk/v8/html/M_Resto_Front_Api_IOperationService_GetWaiterTeams.htm), который даёт возможность получить список всех бригад и [`TryGetWaiterTeamForUser(IUser user)`](https://iiko.github.io/front.api.sdk/v8/html/M_Resto_Front_Api_IOperationService_TryGetWaiterTeamForUser.htm), позволяющий получить бригаду, к кото ...
```
```suggestion
Также в API стали доступны методы [`GetWaiterTeams()`](https://iiko.github.io/front.api.sdk/v8/html/M_Resto_Front_Api_IOperationService_GetWaiterTeams.htm), который даёт возможность получить список всех бригад, и [`TryGetWaiterTeamForUser(IUser user)`](https://iiko.github.io/front.api.sdk/v8/html/M_Resto_Front_Api_IOperationService_TryGetWaiterTeamForUser.htm), позволяющий получить бригаду, в которую назначен сотрудник.
```

zed220 commented on _posts/2022-07-25-tax-rates.md:
```diff
@@ -0,0 +1,12 @@
+- Код ТН ВЭД ЕАЭС - [`OuterEconomicActivityNomenclatureCode`](https://iiko.github.io/front.api.sdk/v8/html/P_Resto_Front_Api_Data_Assortment_IProduct_OuterEconomicActivityNomenclatureCode.htm)
+- Код акцизной марки, который был просканирован при добавлении товара - [`CommodityMark`](https://iiko.github.io/front.api.sdk/v8/html/P_Resto_Front_Api_Data_Orders_IOrderProductItem_CommodityMark.htm)
+
+Также в API был добавлен интерфейс для OuterEconomicActivityNomenclatureCode - [`IOuterEconomicActivityNomenclatureCode`](https://iiko.github.io/front.api.sdk/v8/html/T_Resto_Front_Api_Data_Device_IOuterEconomicActivityNomenclatureCode.htm).
```
Может, написать, что такое OuterEconomicActivityNomenclatureCode?

dterpal replied:
Това́рная номенклату́ра внешнеэкономи́ческой де́ятельности Евразийского экономического союза? Я, конечно, не имею опыта в этих делах, но мне от расшифровки особо понятнее не стало

zed220 replied:
Дело в том, что ты написал наше апишное название, но не расшифровал, что это. Как именно расшифровывать - я не знаю. Нужно узнать у того, кто знает. У тех, кто делал задачу, аналитиков и т.д.

dterpal replied:
Спросил я у Никиты Синко. Он считает, что надо писать "Код УКТЗЄД"

zed220 replied:
Значит, давай так и сделаем.

letolica replied:
`IOuterEconomicActivityNomenclatureCode` — это разве не "Код ТН ВЭД ЕАЭС" из 9 строчки?

dterpal committed:
Update _posts/2022-07-25-waiter-teams.md

Co-authored-by: Zed <pzed220@gmail.com>

letolica commented:
Зачем коммитить с комментарием "Front.Api"? Здесь вся документация про фронтовое апи. Правила комментирования коммитов на гитхаб такие же, как и во внутреннем репозитории: <номер-задачи><пробел><краткое-описание>. Если уж случилось так, что в одном коммите несколько задач, то надо хотя бы номера задач перечислить через запятую или через плюсик.

letolica commented on _posts/2022-07-25-waiter-teams.md:
```diff
@@ -0,0 +1,8 @@
+
+В API V8Preview2 был добавлен интерфейс бригады официантов [`IWaiterTeam`](https://iiko.github.io/front.api.sdk/v8/html/T_Resto_Front_Api_Data_Device_IWaiterTeam.htm).
+
+Также в API стали доступны методы [`GetWaiterTeams()`](https://iiko.github.io/front.api.sdk/v8/html/M_Resto_Front_Api_IOperationService_GetWaiterTeams.htm), который даёт возможность получить список всех бригад, и [`TryGetWaiterTeamForUser(IUser user)`](https://iiko.github.io/front.api.sdk/v8/html/M_Resto_Front_Api_IOperationService_TryGetWaiterTeamForUser.htm), позволяющий получить бригаду, в кот ...
```
Когда пишем новость, для методов не перечисляем параметры, вообще не пишем скобки, достаточно названия. Параметров может быть много, тогда перечисляем, что значит каждый, отдельным списком. Здесь это избыточно.

Также, разрабы плагинов вообще могут быть не в курсе, что за бригады. Белый, Пчёла, Космос, Фил? Надо бы добавить ссылку на документацию. У нас есть примеры таких новостей. Например: https://iiko.github.io/front.api.doc/2019/12/17/mark-order-as-tab.html

dterpal replied:
1.  IOuterEconomicActivityNomenclatureCode, ну да, это интерфейс для "Код ТН ВЭД ЕАЭС", а что не надо говорить об интерфейсе?
2. По поводу комментария - единственное, что я нашел [это](https://ru.iiko.help/articles/#!releasenotes/2022-summer/q/%25D0%2591%25D1%2580%25D0%25B8%25D0%25B3%25D0%25B0%25D0%25B4%25D0%25B0%2520%25D0%25BE%25D1%2584%25D0%25B8%25D1%2586%25D0%25B8%25D0%25B0%25D0%25BD%25D1%2582%25D0%25BE%25D0%25B2/qid/1169616/qp/1)

dterpal committed:
RMS-53640,RMS-53642

Посты о добавлении в API бригад официантов и налоговых ставок, и кодов товаров.

letolica commented on _posts/2022-07-25-waiter-teams.md:
```diff
@@ -3,6 +3,12 @@ title: Бригады официантов
+который даёт возможность получить список всех бригад, и [`TryGetWaiterTeamForUser`](https://iiko.github.io/front.api.sdk/v8/html/M_Resto_Front_Api_IOperationService_TryGetWaiterTeamForUser.htm), 
+позволяющий получить бригаду, в которую назначен сотрудник.
+
+Подробнее о функционировании бригад официантов в [`документации`](https://ru.iiko.help/articles/#!releasenotes/2022-summer/q/%25D0%2591%25D1%2580%25D0%25B8%25D0%25B3%25D0%25B0%25D0%25B4%25D0%25B0%2520%25D0%25BE%25D1%2584%25D0%25B8%25D1%2586%25D0%25B8%25D0%25B0%25D0%25BD%25D1%2582%25D0%25BE%25D0%25B2/qid/1169616/qp/1).
```
Ну ведь у новости есть короткая ссылка. Ну зачем это полотно? Поищи ссылку у подзаголовка и прикрепи её.

letolica commented on _posts/2022-07-25-tax-rates.md:
```diff
@@ -3,10 +3,10 @@ title: Налоговые ставки и коды товаров
-Также в API был добавлен интерфейс для OuterEconomicActivityNomenclatureCode - [`IOuterEconomicActivityNomenclatureCode`](https://iiko.github.io/front.api.sdk/v8/html/T_Resto_Front_Api_Data_Device_IOuterEconomicActivityNomenclatureCode.htm).
\ No newline at end of file
+- Налоговую ставку товара - [`TaxCategory`](https://iiko.github.io/front.api.sdk/v8/html/P_Resto_Front_Api_Data_Assortment_IProduct_TaxCategory.htm);
+- Код УКТЗЄД - [`OuterEconomicActivityNomenclatureCode`](https://iiko.github.io/front.api.sdk/v8/html/P_Resto_Front_Api_Data_Assortment_IProduct_OuterEconomicActivityNomenclatureCode.htm). Также в API был добавлен интерфейс [`IOuterEconomicActivityNomenclatureCode`](https://iiko.github.io/front.api.sdk/v8/html/T_Resto_Front_Api_Data_Device_IOuterEconomicActivityNomenclatureCode.htm);
```
Ну ведь это не код, выходит. А целый объект, внутри которого помимо кода есть другие поля. Надо делать подсписок к этому пункту и описывать все поля твоего интерфейса тут же.

dterpal committed:
RMS-53640,RMS-53642

Исправлена ссылка на документацию бригах официантов и добавлен список параметров для интерфейса IOuterEconomicActivityNomenclatureCode

letolica commented on _posts/2022-07-25-tax-rates.md:
```diff
@@ -0,0 +1,15 @@
+	- [`IsMarked`](https://iiko.github.io/front.api.sdk/v8/html/P_Resto_Front_Api_Data_Device_IOuterEconomicActivityNomenclatureCode_IsMarked.htm) - промаркирован ли товар;
+	- [`AllowSkipMarkScan`](https://iiko.github.io/front.api.sdk/v8/html/P_Resto_Front_Api_Data_Device_IOuterEconomicActivityNomenclatureCode_AllowSkipMarkScan.htm) - возможен ли пропуск сканирования марки;
+	- [`OuterEanCode`](https://iiko.github.io/front.api.sdk/v8/html/P_Resto_Front_Api_Data_Device_IOuterEconomicActivityNomenclatureCode_OuterEanCode.htm) - строка с кодом УКТЗЄД;
+- Код акцизной марки, который был просканирован при добавлении товара - [`CommodityMark`](https://ru.iiko.help/articles/releasenotes/2022-summer).
```
По ссылке, которую ты привел, много новостей. Найди нужный заголовок именно твоей новости, появляется значок "поделиться", нажми на него  скопируй ссылку из адресной строки.
![image](https://user-images.githubusercontent.com/51158040/194860478-31fa030c-9753-4ac4-b310-6f999a0264cc.png)

letolica commented on _posts/2022-07-25-waiter-teams.md:
```diff
@@ -0,0 +1,14 @@
+который даёт возможность получить список всех бригад, и [`TryGetWaiterTeamForUser`](https://iiko.github.io/front.api.sdk/v8/html/M_Resto_Front_Api_IOperationService_TryGetWaiterTeamForUser.htm), 
+позволяющий получить бригаду, в которую назначен сотрудник.
+
+Подробнее о функционировании бригад официантов в [`документации`](https://ru.iiko.help/articles/releasenotes/2022-summer).
```
То же самое. Надо привести не абстрактную ссылку на все новости, а конкретную ссылку на твою новость.

dterpal committed:
RMS-53642

Исправлена ссылка на документацию на гит про CommodityMark

dterpal committed:
RMS-53640

Исправлена ссылка на iiko.help про бригады
